AutomatedTesting/Gem/PythonTests/assetpipeline/TestSuite_Periodic.py

```diff
@@ -10,13 +10,16 @@
 
 from ly_test_tools.o3de.editor_test import EditorTestSuite, EditorSingleTest
 
+
 @pytest.mark.SUITE_periodic
 @pytest.mark.parametrize("project", ["AutomatedTesting"])
 @pytest.mark.parametrize("launcher_platform", ['windows_editor'])
 class TestAutomation(EditorTestSuite):
 
+    @pytest.mark.skip(reason="https://github.com/o3de/o3de/issues/14514")
```

Comment:
Can this be updated to mark these to skip only on Linux? These passed on Windows, and i would like to keep them running on Windows.

Now it's been sandboxed for Windows + Linux.

You can use this, instead
@pytest.mark.skipif(ly_test_tools.LINUX,

Reply:
Conditionally skipping on Linux.